src/vaultRegistry/types.ts

```diff
@@ -0,0 +1,70 @@
+import { ProtocolName, Token } from "src/types";
+import { Address } from "viem";
+
+export type TokenConstant = {
+  chains: number[];
+  address: { [key: string]: Address };
+  name: string;
+  symbol: string;
+  decimals: number;
+  logoURI: string;
+}
+
+export type VaultData = {
+  address: Address;
+  vault: Token;
+  asset: Token;
+  adapter: Token;
+  gauge?: Token;
+  totalAssets: number;
+  totalSupply: number;
+  assetsPerShare: number;
+  assetPrice: number;
+  pricePerShare: number;
+  tvl: number;
+  fees: {
+    deposit: number;
+    withdrawal: number;
+    management: number;
+    performance: number;
+  };
+  depositLimit: number;
```

Comment:
since we're reading this from the contract I'd expect the data type to be `bigint`. Why are we using `number`?

Reply:
Anyone working with this data will also expect it to be `bigint`. I don't think changing it to `number` is a good idea.